pkg/kapp/app/recorded_app.go

```diff
@@ -244,15 +353,24 @@ func (a *RecordedApp) meta() (Meta, error) {
 	}
 
 	app, err := a.coreClient.CoreV1().ConfigMaps(a.nsName).Get(context.TODO(), a.name, metav1.GetOptions{})
```

Comment:
Now that we are continuing [here](https://github.com/vmware-tanzu/carvel-kapp/pull/403#discussion_r795191631), we might want to add this check as well, otherwise it will error out saying configmap is not parseable.